README.md

```diff
@@ -1,393 +1,296 @@
 # Testee
 
-Run your QUnit, Mocha or Jasmine tests from the command line with any browser.
-
-- Runs on all browsers (supporting SocketIO)
-- Many output formats
-- CI integration
-- BrowserStack support
-- Code coverage
-- GruntJS Task
-- Remote URL testing
-
-For questions or discussion checkout our [forums](http://forums.donejs.com/c/testing).
-
-
-## Getting started
-
-You need [NodeJS](http://nodejs.org/) installed in order to use Testee which makes the installation as easy as:
-
-> `npm install -g testee`
-
-`cd` into the main folder of your JavaScript project and run `testee` with your QUnit, Mocha or Jasmine
-test HTML page:
-
-> `testee tests/qunit.html`
-
-
-## Command line testing
-
-The default browser is [PhantomJS](http://phantomjs.org/), just make sure you have it installed anywhere
-on your system.
-
-To run with different local browsers (e.g. Firefox and Safari) use:
-
-> `testee tests/qunit.html --browsers firefox,safari`
-
-Or run multiple tests in multiple browsers:
-
-> `testee tests/unit.html tests/components.html --browsers firefox,safari`
-
-Note that the browser you are using for testing shouldn't be already running (except for PhantomJS which can be started multiple times).
+[![NPM Version][npm-image]][npm-url] [![Build Status][travis-image]][travis-url] [![Dependency Status][david-image]][david-url] [![Gitter Chat][gitter-image]][gitter-url]
 
+> Automated cross-browser testing made easy.
 
-### Command line options
-
-On the command line, you have the following options available:
-
-* `-h`, `--help`: output usage information
-* `-V`, `--version`: output the version number
-* `-b`, `--browsers` `[name]`: A comma separated list of browsers you want to run (default: `phantom`)
-* `-R`, `--root [path|URL]`: The server root path or URL the files are relative to
-* `-p`, `--port` `[port]`: The port to run the server on (default: `3996`)
-* `-r`, `--reporter` `[name]`: The name of the reporter to use (default: `Dot`)
-* `-c`, `--config` `[file]`: Use this JSON or JS configuration file (can be overridden by command line options)
-* `--timeout` `[seconds]`: The per test timeout (in seconds)
-* `--delay` `[ms]`: When running multiple tests, the time to wait for the browser to shut down before starting it with a new page.
-* `-s`, `--server`: Only run the server
-* `--coverage`: Enable code coverage
```

Comment:
This block lengthens an already lengthy document. People used to CLIs typically know to use help screens.